6. Developer Guides/REST API/README.md

```diff
@@ -121,4 +121,7 @@ When calling a production Rocket.Chat server, ensure it is running via HTTPS and
 * [rocket-chat-rest-client](https://github.com/Fab1en/rocket-chat-rest-client)
 
 ### Python
-* [rocketchat_API](https://github.com/jadolg/rocketchat_API)
\ No newline at end of file
+* [rocketchat_API](https://github.com/jadolg/rocketchat_API)
+
+### Ruby
+* [rocketchat-ruby](https://github.com/abrom/rocketchat-ruby)
```

Comment:
@graywolf336 might have been a little misunderstanding here. The link in the PR is to a 'current' API implementation. The older one is https://github.com/int2xx9/ruby-rocketchat